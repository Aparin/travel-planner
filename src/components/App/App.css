h1 {
    color: #27aedb;
    text-align: center;
}

#root {
    margin: center;
    width: 1280px;
}