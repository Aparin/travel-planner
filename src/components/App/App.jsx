import React, { Component } from 'react';
import './App.css';

class App extends Component {
  constructor(props) {
    super(props);
    this.state = {
    };
  }

  render() {
    return (
      <h1>Travel Planner</h1>
    );
  }
}

export default App;
