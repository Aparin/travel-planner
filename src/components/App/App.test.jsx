/* eslint-disable no-undef */
import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import App from './App';

describe('App', () => {
  it('renders App without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  test('App snapshot', () => {
    const component = renderer.create(
      <App />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
