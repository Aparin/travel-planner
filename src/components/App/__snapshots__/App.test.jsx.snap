// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App snapshot 1`] = `
Array [
  <h1>
    Travel Planner
  </h1>,
  <div
    id="leftGroup"
  >
    <a
      href="/help"
      onClick={[Function]}
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary"
        disabled={false}
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "marginBottom": "20px",
            "marginLeft": "100px",
          }
        }
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Как пользоваться
        </span>
      </button>
    </a>
    <input
      className="input"
      id="NewPlaceName"
      onChange={[Function]}
      onKeyUp={[Function]}
      placeholder="Введите название места и нажмите 'Enter'"
      type="text"
      value=""
    />
  </div>,
  <div
    style={
      Object {
        "display": "inline-block",
        "height": "500px",
        "width": "800px",
      }
    }
  />,
]
`;
